htmlelements-matchers/src/main/java/ru/yandex/qatools/htmlelements/matchers/decorators/ConditionMatcherDecorator.java

```diff
@@ -22,16 +25,22 @@ public ConditionMatcherDecorator(Matcher<? super T> matcher, E itemToMatchCondit
     protected boolean matchesSafely(T item) {
         if (condition.matches(itemToMatchCondition)) {
             return matcher.matches(item);
+        } else {
+            Description conditionDescription =
+                    new StringDescription()
+                            .appendValue(itemToMatchCondition)
+                            .appendDescriptionOf(condition);
+            throw new AssumptionViolatedException(
+                    String.format("Condition is not satisfied: %s", conditionDescription));
         }
-
-        Description conditionDescription = new StringDescription();
-        conditionDescription.appendValue(itemToMatchCondition).appendDescriptionOf(condition);
-        throw new AssumptionViolatedException(String.format("Condition is not satisfied: %s", conditionDescription));
     }
 
     @Override
     public void describeTo(Description description) {
-        description.appendDescriptionOf(matcher).appendText(" if ").appendDescriptionOf(condition);
+        description
```

Comment:
I felt it was slightly more readable to have shorter lines with each action on one line.